.github/workflows/solidity.yml

```
@@ -30,6 +32,10 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
+      - name: Set cache key
+        id: set-cache-key
+        run: echo "cache-key=ci-solidity-${{ hashFiles('**/*.sol') }}" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:62: Double quote to prevent globbing and word splitting

(shellcheck)